gulp-tasks/wfContributors.js

```diff
@@ -81,22 +81,25 @@ function buildIndex(contributors) {
  */
 function buildIndividualPages(contributors) {
   gutil.log(' ', 'Building individual pages of all contributors...');
-  const files = wfHelper.getFileList(global.WF.src.content, ['**/*.md']);
-  const filesByAuthor = wfHelper.splitByAuthor(files);
-  const keys = Object.keys(contributors);
-  keys.forEach(function(key) {
-    if (!(key in filesByAuthor)) {
-      return;
-    }
-    const contributor = contributors[key];
-    filesByAuthor[key].sort(wfHelper.publishedComparator);
-    const context = {
-      id: key,
-      contributor: contributor,
-      articles: filesByAuthor[key],
-    };
-    const dest = DEST_ARTICLE_LIST.replace('{{key}}', key);
-    wfTemplateHelper.renderTemplate(TEMPLATE_ARTICLE_LIST, context, dest);
+  global.WF.langs.forEach(function(lang) {
+    const folder = global.WF.src.content.replace('/en', '/'.concat(lang));
+    const files = wfHelper.getFileList(folder, ['**/*.md']);
+    const filesByAuthor = wfHelper.splitByAuthor(files);
+    const keys = Object.keys(contributors);
+    keys.forEach(function(key) {
+      if (!(key in filesByAuthor)) {
+        return;
+      }
+      const contributor = contributors[key];
+      filesByAuthor[key].sort(wfHelper.publishedComparator);
+      const context = {
+        id: key,
+        contributor: contributor,
+        articles: filesByAuthor[key],
+      };
+      const dest = DEST_ARTICLE_LIST.replace('{{key}}', key);
```

Comment:
`const DEST_ARTICLE_LIST = './src/content/en/resources/contributors/{{key}}.md';` - which means it's writing all of the files to the EN directory, not the individually translated ones.

Reply:
Need improvement because produce fs-style links (note **lang** in path):
Example: `/web/resources/contributors/src/content/it/fundamentals/performance/resource-prioritization`

Reply:
@petele This PR produce a "contribution page" per person for every language.
Ex: https://developers.google.com/web/resources/contributors/petelepage ( `src/content/en/resources/contributors/petelepage.md` ). The previous implementation don't generate a "contribution page" for not-English contributions.